Jenkinsfile

```diff
@@ -358,177 +326,155 @@ gcloud container clusters create ${clusterName} \
                                     }
                                 }
-                                finally {
-                                    container('az') {
-                                        sh "az login --service-principal -u \$AZURE_CLIENT_ID -p \$AZURE_CLIENT_SECRET -t \$AZURE_TENANT_ID"
-                                        sh "az account set -s \$AZURE_SUBSCRIPTION_ID"
-                                        sh "az network dns zone delete --yes --name ${dnsZone} --resource-group ${resourceGroup} || :"
-                                        sh "az aks delete --yes --name ${clusterName} --resource-group ${resourceGroup} --no-wait || :"
-                                    }
-                                    deleteGlueRecords(dnsPrefix, parentZone, parentZoneResourceGroup)
-                                }
                             }
                         }
                     }
                 }
-            }
-        }
-    }
 
-    platforms.failFast = true
-    parallel platforms
-
-    stage('Release') {
-        node(label) {
-            if (env.TAG_NAME) {
-                timeout(time: 30) {
-                    dir('src/github.com/bitnami/kube-prod-runtime') {
-                        unstash 'src'
-                        withGo() {
-                            unstash 'release-notes'
-
-                            sh "make dist VERSION=${TAG_NAME}"
-
-                            withCredentials([
-                                usernamePassword(credentialsId: 'github-bitnami-bot', passwordVariable: 'GITHUB_TOKEN', usernameVariable: ''),
-                                [
-                                $class: 'AmazonWebServicesCredentialsBinding',
-                                credentialsId: 'jenkins-bkpr-releases',
-                                accessKeyVariable: 'AWS_ACCESS_KEY_ID',
-                                secretKeyVariable: 'AWS_SECRET_ACCESS_KEY',
-                                ]
-                            ]) {
-                                sh "make publish VERSION=${TAG_NAME}"
+                parallel platforms
+
+                stage('Release') {
+                    if (env.TAG_NAME) {
+                        dir("${env.WORKSPACE}/src/github.com/bitnami/kube-prod-runtime") {
+                            withGo() {
+                                withCredentials([
+                                    usernamePassword(credentialsId: 'github-bitnami-bot', passwordVariable: 'GITHUB_TOKEN', usernameVariable: 'GITHUB_USER'),
+                                    [$class: 'AmazonWebServicesCredentialsBinding', credentialsId: 'jenkins-bkpr-releases', accessKeyVariable: 'AWS_ACCESS_KEY_ID', secretKeyVariable: 'AWS_SECRET_ACCESS_KEY']
+                                ]) {
+                                    withEnv(["PATH+ARC=${tool 'arc'}"]) {
+                                        sh "make dist VERSION=${TAG_NAME}"
```

Comment:
It would probably look simpler if we smooshed these 3 `withEnv/sh` blocks into a single `withEnv/sh` that installed all 3 tools and ran a single `make dist release-notes publish`, rather than trying to be "minimal" at each step.
(or maybe we still need 3 `make` invocations for side-effect ordering?)

Reply:
fixed!

Reply:
Did `make release-notes` get lost?  Was that intentional?

Reply:
No not lost. `make release-notes` is a dependency for `make release-notes`